example/CONTRIBUTING.md

```diff
@@ -1,12 +1,12 @@
-Thank you for contributing {{.RepoName}}!
+Thank you for contributing to {{.RepoName}}!
 =========================================
 
-English and Japanese are available.  Japanese is better but English is also OK.
+You can speak in Japanese or English.  Japanese is better but English is also OK.
```

Comment:
this isn't better grammar either though.

If you look the rest is written in third-person active, this new sentence is first-person neutral.

Reply:
My suggestion, rewrite the entire sentence to something like:

Please use either Japanese or English, Japanese is the preferred language.